flake.nix

```
@@ -0,0 +1,17 @@
+{
+  description = "shabitica flake";
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/21.05";
+
+  outputs = { nixpkgs, ... }: {
+    nixosModules.default = { lib, config, ... }: {
+      imports = [
+        (import ./default.nix {
+          inherit lib config;
+          pkgs = import nixpkgs {
+            inherit (config.nixpkgs.localSystem) system;
+          };
+        })
+      ];
+    };
```

The `default.nix` in the project root is for supporting Hydra channels, so you can just boil it down to the module entrypoint:

```suggestion
    nixosModules.default.imports = [ ./modules ];
```